source/installation/modify-system-security.rst

```diff
@@ -32,27 +23,52 @@ Modify System Security
          Policy deny_unknown status:     allowed
          Max kernel policy version:      31
 
+#. Disable `SELinux`_ on the Open OnDemand host you intend on running the web
+   server on. 
 
-   RHEL/CentOS 6/7
-      .. code-block:: sh
+   .. note::
+
+      If SELinux is not disabled on the web server host machine this can lead
+      to permission errors when running the web applications. SELinux may also
+      need to be disabled on the web server if you want to take advantage of
+      certain software in your web applications such as being able to access a
+      Lustre file system.
 
-         $ sudo chmod -R 757 /etc/selinux/config  
 
-   Using your favorite text editor open the /etc/selinux/config file and set the SELINUX mode to disabled 
+   **Option 1: Disable SELinux Temporarily**
```

Comment:
Using `setenforce 0` should not be presented as a separate option from editing `/etc/selinux/config`. They should be used together.